.travis.yml

```
@@ -13,7 +14,6 @@ os:
 script:
   - travis_retry npm run test-only
```

This PR has fixed our flakiness in CI!

```suggestion
  - npm run test-only
```